[SIEMINT-85] DDS: Trend Micro Vision One XDR: Crawler Integration v1.0.0 (#18208)
DataDog/integrations-core: trend_micro_vision_one_xdr/README.md

## Overview

> [Trend Micro Vision One XDR][1] collects and automatically correlates data across multiple security layers - email, endpoint, server, cloud workload, and network. This allows for faster detection of threats and improved investigation and response times through security analysis.

Reviewer: ```suggestion
[Trend Micro Vision One XDR][1] collects and automatically correlates data across multiple security layers: email, endpoint, server, cloud workload, and network. This allows for faster detection of threats and improved investigation and response times through security analysis.
```

Author: Done

This integration ingests the following logs:

- Workbench Alerts
- Observed Attack Techniques

> This integration seamlessly collects all the above listed logs, channeling them into Datadog for analysis. Leveraging the built-in logs pipeline, these logs are parsed and enriched, enabling effortless search and analysis. The integration provides insight into workbench alerts and observed attack techniques through the out-of-the-box dashboards.

Reviewer: ```suggestion
This integration collects all the above listed logs and sends them to Datadog for analysis. Datadog uses the built-in logs pipeline to parse and enrich these logs, enabling effortless search and analysis. The integration provides insight into workbench alerts and observed attack techniques through the out-of-the-box dashboards.
```

Author: Done

2. Generate a new authentication token. Click **Add API key**. Specify the settings of the new API key.
    - Name: A meaningful name that can help you identify the API key
    - Role: The user role assigned to the key. Select **SIEM** from dropdown.
    - Expiration time: The time the API key remains valid.
    - Status: Whether the API key is enabled.
    - Details: Extra information about the API key.
3. Click **Add**.
4. Copy and store the authentication token in a secure location.

#### Trend Micro Vision One XDR DataDog Integration Configuration

Configure the Datadog endpoint to forward Trend Micro Vision One XDR logs to Datadog.

1. Navigate to `Trend Micro Vision One XDR`.
2. Add your Trend Micro Vision One XDR credentials.

| Trend Micro Vision One XDR Parameters | Description                                                  |
| ------------------------------------- | ------------------------------------------------------------ |
| Host Region                           | The Region of your Trend Micro Vision One XDR Console        |
| API Key                               | The API Key of your Trend Micro Vision One XDR Console       |

## Data Collected

### Logs

> The Trend Micro Vision One XDR integration collects and forwards Workbench Alerts and Observed Attack Techniques logs to Datadog.

Reviewer: Is there any setup required here?

Author: We have added the setup configurations steps above. Can you please let us know the exact requirement here?

## Overview

[Trend Micro Vision One XDR][1] collects and automatically correlates data across multiple security layers: email, endpoint, server, cloud workload, and network. This allows for faster detection of threats and improved investigation and response times through security analysis.

This integration ingests the following logs:

- Workbench Alerts

> - Observed Attack Techniques

Reviewer 2: Please provide a description of what is contained in each endpoint

Author: Done

> - Workbench Alerts: This endpoint contains information about all the standalone alerts triggered by detection models.

Reviewer 3: [nit] Can we bold "Workbench Alerts "and "Observed Attack Techniques"

Author: Done

> This integration collects all the above listed logs and sends them to Datadog for analysis. Datadog uses the built-in logs pipeline to parse and enrich these logs, enabling effortless search and analysis. The integration provides insight into workbench alerts and observed attack techniques through the out-of-the-box dashboards.

Reviewer 2: Please add that we also offer out of the box detection rules for Ccloud SIEM customers.

Author: Done

> This integration collects all the above listed logs and sends them to Datadog for analysis. Datadog uses the built-in logs pipeline to parse and enrich these logs, enabling effortless search and analysis. The integration provides insight into workbench alerts and observed attack techniques through the out-of-the-box dashboards. Also, This integration provides out of the box detection rules.

Reviewer 3: [nit] This integration collects logs from the sources listed above and sends them to Datadog for analysis with our Log Explorer and Cloud SIEM products.

Author: Done

## Setup

### Configuration

#### Create API KEY from Trend Micro Vision One XDR

> 1. On the Trend Vision One console, go to **Administration > API Keys** .

Reviewer 3: On > In

Author: Done

> 2. Generate a new authentication token. Click **Add API key**. Specify the settings of the new API key.

Reviewer 3: "Specify the settings of the new API key." > "Specify the settings of the new API key with the following:"

Author: Done

>     - Name: A meaningful name that can help you identify the API key

Author: Done

    - Role: The user role assigned to the key. Select **SIEM** from dropdown.
    - Expiration time: The time the API key remains valid.
    - Status: Whether the API key is enabled.
    - Details: Extra information about the API key.
3. Click **Add**.
4. Copy and store the authentication token in a secure location.

#### Trend Micro Vision One XDR DataDog Integration Configuration

Configure the Datadog endpoint to forward Trend Micro Vision One XDR logs to Datadog.

1. Navigate to `Trend Micro Vision One XDR`.

> 2. Add your Trend Micro Vision One XDR credentials.

Author: Updated message with more specific details

## Overview

> [Trend Micro Vision One XDR][1] collects and automatically correlates data across multiple security layers: email, endpoint, server, cloud workload, and network. This allows for faster detection of threats and improved investigation and response times through security analysis.

Reviewer 3: [Nit] - Please remove the double `and` sentence, maybe something like:  `This enables faster threat detection and enhances investigation and response times through improved security analysis.`

Author: Done

This integration ingests the following logs:

- **Workbench Alerts**: This endpoint contains information about all the standalone alerts triggered by detection models.
- **Observed Attack Techniques**: This endpoint contains information about observed attack techniques from Detections, Endpoint Activity, Cloud Activity, Email Activity, Mobile Activity, Network Activity, Container Activity, and Identity Activity data sources.

This integration collects logs from the sources listed above and sends them to Datadog for analysis with our Log Explorer and Cloud SIEM products

> * https://docs.datadoghq.com/logs/explorer/

Reviewer 3: Can we make these hyperlinks on the words instead of direct links? [Log Explorer][3], [Cloud SIEM][4]

Author: Done

* https://www.datadoghq.com/product/cloud-siem/

## Setup

### Configuration

#### Create API KEY from Trend Micro Vision One XDR

> 1. In the Trend Vision One console, go to **Administration > API Keys**.

Reviewer 3: go to > visit

Author: Done

> 1. In the Trend Vision One console, go to **Administration > API Keys** .

Reviewer 3: [nit] Can we please remove the extra space in front of . please?

Author: Done
